core/src/main/scala/org/apache/spark/ExecutorAllocationClient.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark
 
-import org.apache.spark.scheduler.ExecutorDecommissionInfo
+import org.apache.spark.scheduler.ExecutorDecommissionReason
```

Comment:
Why are we renaming this?

Reply:
+1, I would argue against un-necessary renaming even if it seems a bit "unnatural". It creates un-necessary diff noise. 

To me "Info" and "Reason" are both similar: They both portend "additional information".

Reply:
I guess @Ngone51 is trying to follow `TaskEndReason`.

Reply:
Thank @dongjoon-hyun for the clarification. In addition to follow the style of `TaskEndReason`, I acutally also want to handle the decommission info/reason in the similar way of `TaskEndReason`.